src/hooks/useActiveProject.tsx

```diff
@@ -76,8 +80,8 @@ export const ActiveProjectContextProvider = ({
   useEffect(() => {
     if (
       hookReturnValue.activeProject?.id || // active project already set
-      !projectsResult.data?.length || // wait for projects endpoint to return data
-      !useMeResult.data?.length || // wait for subjects endpoint to return data
+      projectLoading || // wait for projects endpoint
```

Comment:
The presence of data should not dictate when this hook begins computing an activeProject given it is valid to arrive here without subjects or projects associated to a user. For everything to flow correctly we should instead attempt to compute the hook result if the queries we depend on have settled one way or another.